### ISSUE TITLE
Jcenter removal

### ISSUE THREAD

**Issue by matt-dalton** [RevenueCat/react-native-purchases#368]
- [x] I have updated Purchases SDK to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/RevenueCat/react-native-purchases/blob/main/CONTRIBUTING.md)
- [x] I have searched the [Community](https://community.revenuecat.com)
- [x] I have read [docs.revenuecat.com](https://docs.revenuecat.com/)
- [x] I have searched for [existing Github issues](https://github.com/RevenueCat/react-native-purchases/issues)

**Describe the bug**
Are there any plans to remove jcenter from the android repos? On build, our project is now throwing an error since it's now deprecated/removed:
```
WARNING:: Please remove usages of `jcenter()` Maven repository from your build scripts and migrate your build to other Maven repositories.
This repository is deprecated and it will be shut down in the future.
See http://developer.android.com/r/tools/jcenter-end-of-service for more information.
Currently detected usages in: project project ':react-native-purchases',
```

1. Environment
   1. Platform: React Native/android
   2. SDK version: 4.6.0
   3. OS version: macOs 12.2.1
   4. Xcode/Android Studio version: AS 2021.2.1
   5. React Native version: 0.63
   6. SDK installation (CocoaPods + version or manual): automatic
   7. How widespread is the issue. Percentage of devices affected. All new builds
2. [Debug logs](https://docs.revenuecat.com/docs/debugging) that reproduce the issue
3. Steps to reproduce, with a description of expected vs. actual behavior
4. **Other information** (e.g. stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc.)

**Additional context**

**Comment by vegaro** [comment]
Thanks for opening @matt-dalton 

We are going to remove the usages for the next major

**Comment by matt-dalton** [comment]
Amazing thankyou. Would it be possibly to get this into a patch on this release?

**Comment by NachoSoto** [comment]
Version 4.6.1 will have this fix: #372
